### PR TITLE
feat: lender profile summary component added

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Summary component added to lender profile

Requirements: https://kiva.atlassian.net/browse/MP-402

Screenshot 2024-07-22 at 11 22 06 a m

Screenshot 2024-07-22 at 11 21 54 a m

### CODE REVIEW

File: src/components/LenderProfile/LenderSummary.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<template>
+	<div>
+		<h3>Kiva Lender {{ lenderName }}</h3>
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:3]: We tend to put `data-hj-suppress` on elements with lender name.

File: src/components/LenderProfile/LenderSummary.vue [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+						v-for="(value, key, index) in lenderSummary"
+						:key="index"
+						class="tw-py-1.5 tw-flex tw-gap-x-2"
+					>
+						<dt class="tw-font-medium tw-capitalize tw-basis-1/3 md:tw-basis-1/4">
+							{{ key }}:
+						</dt>
+						<dd class="tw-basis-2/3 lg:tw-basis-3/4">
+							<template v-if="key=='checkout'">
+								<a
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:23]: Click analytics on this anchor?

Comment (roger-in-kiva) [reply src/components/LenderProfile/LenderSummary.vue:25]: This is a nice call but I don't have specifications.
@emuvente can you add a new ticket for analytics events?

**Review by dyersituations — APPROVED** [review]

File: src/components/LenderProfile/LenderSummary.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+									:href="value"
+									target="_blank"
+								>
+									{{ value }}
+								</a>
+							</template>
+							<template v-else>
+								{{ value }}
```
Comment (mcstover) [inline src/components/LenderProfile/LenderSummary.vue:33]: I think these values need to be supressed as well. We shouldn't be shipping off Lender location and occupation to 3rd parties...

File: src/components/LenderProfile/LenderSummary.vue (hunk 2 above)
Comment (emuvente) [reply src/components/LenderProfile/LenderSummary.vue:25]: MP-570

**Comment by roger-in-kiva** [comment]
@emuvente Lender image is added.

Screenshot 2024-07-23 at 10 25 07 a m
Screenshot 2024-07-23 at 10 25 16 a m

**Review by dyersituations — APPROVED** [review]

File: src/graphql/query/lenderPublicProfile.graphql [hunk 4]
```diff
@@ -4,6 +4,7 @@ query LenderPublicProfile ($publicId: String!) {
 			id
 			name
 			loanCount
+			memberSince
 			image {
 				id
 				url
```
Comment (emuvente) [inline src/graphql/query/lenderPublicProfile.graphql:10]: The default url gives a 100x100 sized image, so let's change this to `url(customSize: "w200h200")` to match the size in the old-stack lender page.

Comment (roger-in-kiva) [reply src/graphql/query/lenderPublicProfile.graphql]: Done.

**Review by emuvente — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**